### DISCUSSION TITLE
Reviewer Settings for CFP

### DISCUSSION THREAD

**Discussion by TosinDoreen** [pretalx/pretalx#1719] [General]
Hello everyone, I need assistance setting-up accurate reviewer settings for my CFP on the Pretalx platform. 

Under settings, I have put the necessary avenues for reviewing the CFPs to require text and score. However, when I sign up as a reviewer on another account, I cannot see the review forms where I am to input a score, or even give feedback in text format. 

Does this have to do with the fact that the review phase was set for April? Could that be the reason for the actual review parameters not showing yet?

I need clarity, as this is my first time using Pretalx. 

Looking forward to receiving a response

**Comment by rixx** [comment]
There are many reasons why a review account can't review proposals, for example:

- there is no current review phase
- the current review phase doesn't permit to review proposals
- you don't have reviewer permissions to your event
- your event requires reviewers to be assigned proposals, but you haven't been assigned any proposals
